share/completions/black.fish

```diff
@@ -0,0 +1,23 @@
+complete -c black -s c -l code -d 'Code to format'
```

Comment:
The `-c` is not necessary. (Not sure if we should care about that?)

Reply:
nevermind, let's keep it for compatibility, see also https://github.com/fish-shell/fish-shell/pull/8113#discussion_r667439277